systemtests/bin/upload-symbols.py

```diff
@@ -82,7 +83,10 @@ def upload_symbols(ctx, base_url, auth_token, symbolsfile):
                         headers={"auth-token": auth_token},
                         timeout=CONNECTION_TIMEOUT,
                     )
-                if resp.status_code != 201:
+                if resp.status_code == 429:
+                    click.echo(click.style("429--sleeping for 5", fg="yellow"))
+                    time.sleep(5)
+                elif resp.status_code != 201:
```

Comment:
I should add explicit handling for 403 here and in `upload-symbols-by-download.py`.

Also, it'd be better if the script took an expected status code so it could "yay" or "nay" based on the expectation since we expect uploading with a bad token to fail.